src/components/AddTaskCard/AddTaskCard.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import { sanitizeText } from "src/utils/sanitizeText";
+import "./AddTaskCard.scss";
+
+function AddTaskCard({ onCreateTask }) {
+  const [taskTitle, setTaskTitle] = useState("");
+  const sanitizedTaskTitle = sanitizeText(taskTitle);
+  const isTaskButtonDisabled = sanitizedTaskTitle === "";
```

Comment:
Please use null instead of empty string.

Reply:
@rsa-wtag as the taskTitle state is used as a string and later some regex operations are applied to that string, setting them null actually breaks the whole code, as regex operations can't be done on null. 

And comparing the `sanitizedTaskTitle` to `null` doesn't actually gives the same result as comparing it with an empty string. So, that also gives unwanted output.

That's why for this case empty strings are kept as it is instead of using null.